README.md

```diff
@@ -6,11 +6,16 @@ For general questions about using the virtual environments or writing your Actio
 ## Available Environments
 | Environment | YAML Label | Included Software | Latest Release & Rollout Progress |
 | --------------------|---------------------|--------------------|---------------------|
+| Ubuntu 20.04 | `ubuntu-20.04` | [ubuntu-20.04] | *Coming soon* |

```

Comment:
@alepauly , @al-cheb , do you know how to create link like `https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=ubuntu18&badge=1` for Ubuntu 20?

Looks like Ubuntu 20 is not there: https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status

Reply:
@lupine86 might know how to create a link.

Reply:
Fixed query so that Ubuntu20 shows up now.
https://github.com/actions/virtual-environments-deployment-status/commit/fe4fb7e91905ee457f6755404c9cab2840071995

Reply:
Thank you! Everything works as expected.